examples/oneapi/dpc/daal_lnx.lst

```diff
@@ -18,5 +18,7 @@
 ##     Intel(R) Data Analytics Acceleration Library examples list
 ##******************************************************************************
 
-DAAL = pca_cor_dense_batch      \
+DAAL = knn_cls_brute_force_dense_batch \
+       pca_cor_dense_batch             \
```

Comment:
Should we enumerate it in alphabetical order for clarity? Smth like:
```
DAAL = homogen_table_builder                \
             knn_cls_brute_force_dense_batch \
             pca_cor_dense_batch
```

Reply:
Agree